learn/sherif/main_test/src/main/java/frc/robot/subsystems/ShooterController.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.helpers.MockableSparkMax;
+import frc.robot.interfaces.CANSparkMaxInterface;
+
+public class ShooterController extends SubsystemBase {
+    private final CANSparkMaxInterface leftSide;
+    private final CANSparkMaxInterface rightSide;
+    private final boolean useFollower;
```

Comment:
This variable is unnecessary. It's given a value from the constructor, but it is never actually used anywhere.